From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 3 Aug 2026 00:58:16 +0200
Subject: ARM: dts: allwinner: sun8i-a83t-tbs-a711: Add the accelerometer mount
 matrix

The bma250 is mounted a quarter turn from the display: its X axis is
the screen's Y and its Y axis is the screen's -X. Measured against
gravity in five poses, with the tablet in its normal landscape
orientation.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
index 111111111111..222222222222 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
@@ -343,6 +343,9 @@ &i2c1 {
 	accelerometer@18 {
 		compatible = "bosch,bma250";
 		reg = <0x18>;
+		mount-matrix = "0", "-1", "0",
+			       "1", "0", "0",
+			       "0", "0", "1";
 		interrupt-parent = <&pio>;
 		interrupts = <7 10 IRQ_TYPE_EDGE_RISING>; /* PH10 / EINT10 */
 		vdd-supply = <&reg_dcdc1>;
-- 
Armbian

